templates/countries.json

```diff
@@ -8,14 +8,17 @@
   "A": [
     {
       "name": "Africa",
+      "codes": ["ZA", "CM", "CD", "SN", "KE", "GH"],
```

Comment:
I think you don't use those anymore ?

Reply:
We want those for the GeoIP detection to provide the proper button, like we do for other NROs. For some reason these were missing from Africa, so I just squeezed them in now.